src/stacksUtils/crossContractDataMapping.ts

```diff
@@ -192,6 +201,19 @@ export function contractAssignedChainIdToKnownChain(
   assertExclude(resPossibilities, KnownChainId.Runes.Mainnet)
   assertExclude(resPossibilities, KnownChainId.Runes.Testnet)
 
+  if (chainId === 19n) {
+    return [KnownChainId.Solana.Mainnet, KnownChainId.Solana.Testnet]
+  }
+  assertExclude(resPossibilities, KnownChainId.Solana.Mainnet)
+  assertExclude(resPossibilities, KnownChainId.Solana.Testnet)
+
+  // TBD
+  if (chainId === 999n) {
```

Comment:
the chainId for Tron above is 20n

<img width="301" alt="image" src="https://github.com/user-attachments/assets/36f16e0c-a7b4-460b-a55b-0bc57718a011" />